src/gtc/gtir.py

```diff
@@ -234,14 +234,23 @@ def _no_write_and_read_with_horizontal_offset(
         return values
 
 
+class Argument(Node):
+    name: Str
+    is_keyword: bool
+    default: Str
+
+
 class Stencil(LocNode, SymbolTableTrait):
     name: Str
-    # TODO(havogt) deal with gtscript externals
+    api_signature: List[Argument]
     params: List[Decl]
```

Comment:
I think the information of the `api_signature` should be integrated into params at some point. Either by reflecting the structure of the signature in the values or in the structure of the member (list of arguments + dict of keyword arguments). I would leave a small todo comment.

Reply:
I'll do that in a follow-up: https://github.com/GridTools/gt4py/issues/710.